hudi-hadoop-common/src/test/java/org/apache/hudi/common/table/TestHoodieTableConfig.java

```diff
@@ -63,7 +61,7 @@ public class TestHoodieTableConfig extends HoodieCommonTestHarness {
   @BeforeEach
   public void setUp() throws Exception {
     initPath();
-    storage = HoodieStorageUtils.getStorage(basePath, HadoopFSUtils.getStorageConf(new Configuration()));
+    storage = HoodieStorageUtils.getStorage(basePath);
```

Comment:
Let's create a util method in test utils only.  Production code should not use empty `Configuration`.

Reply:
HoodieParquetDataBlock does. So does InProcessTimeGenerator.

Reply:
https://issues.apache.org/jira/browse/HUDI-7731